.github/workflows/main.yml

```diff
@@ -303,6 +303,7 @@ jobs:
 
               # This allows us to start Connect separately in our own docker container
               CONNECT_SERVER: "http://localhost:3939"
+              IMAGE_OS: "ubuntu22"
```

Comment:
`IMAGE_OS` should be something like `rstudio/connect:ubuntu22`. It's `TARGET_OS` that contains just the distribution.